src/main/java/com/microfocus/application/automation/tools/results/parser/nunit3/NUnit3ReportParserImpl.java

```diff
@@ -0,0 +1,82 @@
+/*
```

Comment:
Wrong copyright notice

Reply:
Without the EntIt, line 2

Reply:
What do you mean? I missed the Entlt? It's there.